charts/gsp-cluster/dashboards/concourse.json

```diff
@@ -882,6 +882,101 @@
         "align": false,
         "alignLevel": null
       }
+    },
+    {
+      "cacheTimeout": null,
+      "colorBackground": true,
+      "colorValue": false,
+      "colors": [
+        "#299c46",
+        "rgba(237, 129, 40, 0.89)",
+        "#d44a3a"
+      ],
+      "datasource": "Prometheus",
+      "format": "short",
+      "gauge": {
+        "maxValue": 100,
+        "minValue": 0,
+        "show": false,
+        "thresholdLabels": false,
+        "thresholdMarkers": true
+      },
+      "gridPos": {
+        "h": 6,
+        "w": 8,
+        "x": 8,
+        "y": 14
+      },
+      "id": 33,
+      "interval": null,
+      "links": [],
+      "mappingType": 2,
+      "mappingTypes": [
+        {
+          "name": "value to text",
+          "value": 1
+        },
+        {
+          "name": "range to text",
+          "value": 2
+        }
+      ],
+      "maxDataPoints": 100,
+      "nullPointMode": "connected",
+      "nullText": null,
+      "options": {},
+      "postfix": "",
+      "postfixFontSize": "50%",
+      "prefix": "",
+      "prefixFontSize": "50%",
+      "rangeMaps": [
+        {
+          "from": "null",
+          "text": "OK",
+          "to": "null"
+        },
+        {
+          "from": "0",
+          "text": "OK",
+          "to": "0"
+        },
+        {
+          "from": "1",
+          "text": "NOT OK",
+          "to": "10"
+        }
+      ],
+      "sparkline": {
+        "fillColor": "rgba(31, 118, 189, 0.18)",
+        "full": false,
+        "lineColor": "rgb(31, 120, 193)",
+        "show": false
+      },
+      "tableColumn": "",
+      "targets": [
+        {
+          "expr": "sum(increase(concourse_builds_finished{team=\"main\", pipeline=\"cd-smoke-test\", status=~\"failed|errored\"}[15m]))",
```

Comment:
I think maybe we shouldn't include `errored` in this expr.

Errored generally means "pipeline is misconfigured" or "failed to get secrets" ... and since we have a seperate panel explicitly for tracking `errored`, having this come combine the two could make it hard to explain.

Reply:
Suppose that `errored` is removed from the expression. If the pipeline is misconfigured, Canary smoke test status panel will show a green colour "OK". This means that it won't tell us that Canary smoke test is not running because the pipeline is misconfigured. Are you okay with this scenario?

Reply:
yes ... I believe we are trying to detect when the tooling we have provided for CI/CD is unable to perform it's task ... not when the cd-smoke-test pipeline itself is misconfigured.

The other graph of job errors will be useful for identifying trends in errored jobs.

Reply:
Thanks. I have removed `errored` from the expression.